CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## Pending
 
+- Fixes an issue when building Carthage framework failed due to a missing file in TABTestKit carthage project.
+- Adds a step script for building carthage framework in Travis CI. This should report to CI if the TABTestKit can be build as a Carthage framework.
+- Adds a step script for linting cocoapods project in Travis CI. This should report to CI if the TABTestKit can be safe to be published as Cocoapods framework.
```

Comment:
I think we might be able to remove the # Pending section and move down below to 1.6.0? Sorry didn't notice this before.